src/briefcase/platforms/linux/system.py

```diff
@@ -0,0 +1,987 @@
+import gzip
+import os
+import re
+import subprocess
+import sys
+from pathlib import Path
+from typing import List
+
+from briefcase.commands import (
+    BuildCommand,
+    CreateCommand,
+    PackageCommand,
+    PublishCommand,
+    RunCommand,
+    UpdateCommand,
+)
+from briefcase.config import AppConfig
+from briefcase.exceptions import BriefcaseCommandError, UnsupportedHostError
+from briefcase.integrations.docker import Docker, DockerAppContext
+from briefcase.integrations.subprocess import NativeAppContext
+from briefcase.platforms.linux import (
+    ARCH,
+    DEBIAN,
+    RHEL,
+    DockerOpenCommand,
+    LinuxMixin,
+    LocalRequirementsMixin,
+    parse_freedesktop_os_release,
+)
+
+
+class LinuxSystemPassiveMixin(LinuxMixin):
+    # The Passive mixin honors the Docker options, but doesn't try to verify
+    # Docker exists. It is used by commands that are "passive" from the
+    # perspective of the build system (e.g., Run).
+    output_format = "system"
+    supported_host_os = {"Darwin", "Linux"}
+    supported_host_os_reason = (
+        "Linux system projects can only be built on Linux, or on macOS using Docker."
+    )
+
+    @property
+    def use_docker(self):
+        # The system backend doesn't have a literal "--use-docker" option, but
+        # `use_docker` is a useful flag for shared logic purposes, so evaluate
+        # what "use docker" means in terms of target_image.
+        return bool(self.target_image)
+
+    @property
+    def linux_arch(self):
+        # Linux uses different architecture identifiers for some platforms
+        return {
+            "x86_64": "amd64",
+        }.get(self.tools.host_arch, self.tools.host_arch)
```

Comment:
I was able to build, install, and run helloworld from a `deb` using Debian 11 on `aarch64` with this change.
```suggestion
        return {
            "x86_64": "amd64",
            "aarch64": "arm64",
        }.get(self.tools.host_arch, self.tools.host_arch)
```